### ISSUE TITLE
Missing Layers in Search Space for NASBench-201

### ISSUE THREAD

**Issue by NeoChaos12** [automl/NASLib#86]
Hi everyone,

I was comparing the code for the search space in the [original repo](https://github.com/D-X-Y/AutoDL-Projects/tree/69ca086) (as per the [changelog](https://github.com/D-X-Y/AutoDL-Projects/blob/main/CHANGE-LOG.md)) with the NASLib implementation since we were observing consistently worse performance (down by 5-6 percentage points) when trying to re-create the top-10 or even top-100 architectures' performance and noticed that, in the NASLib implementation [here](https://github.com/automl/NASLib/blob/047439805abb73802d10ad7e987b57a7b1787b9c/naslib/search_spaces/nasbench201/graph.py#L112-L120), when compared to the corresponding original code [here](https://github.com/D-X-Y/AutoDL-Projects/blob/69ca0860aa379bafe488283a1bca59439528c9e7/lib/models/cell_infers/tiny_network.py#L32-L34), there is a missing BatchNorm + ReLU before the global adaptive pooling layer. Interestingly, I did not see any mention of these layers in the paper on [arXiv](https://arxiv.org/abs/2001.00326). But introducing them by making the following change to the aboive-mentioned NASLib code instantly re-created the expected top-10 model performance:

```python
# post-processing
  self.edges[edge_names["postproc"]].set('op', ops.Sequential(
        nn.BatchNorm2d(channels[-1]),
        nn.ReLU(inplace=False),
        nn.AdaptiveAvgPool2d(1),
        nn.Flatten(),
        nn.Linear(channels[-1], self.num_classes)
  ))
```

**Comment by arberzela** [comment]
Merged to Develop
